index.html

```diff
@@ -42,23 +42,195 @@ <h1>I'm Chrispaix Glad to see you!</h1>
+                    <ul>
+                        <li>Back End dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="read-more">
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </div>
+                <div class="tech-used">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="button">
+                    <button type="button">See Project</button>
                 </div>
             </div>
         </div>
     </section>
 
+    <!--****ABOUT MYSELF****-->
+    <section class="about-me">
+        <div class="container">
+            <div class="aboutme-title">
+                <h2>About Myself</h2>
+            </div>
+            <div class="aboutme-description">
+                <p>
+                    Hello I’m a software developer! I can help you build a product ,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <span>Let's connect</span>
+                <div class="social-media">
+                    <ul>
+                        <li><a href="#"><i class="icon-item fa fa-twitter"></i></a></li>
+                        <li><a href="#"><i class="icon-item fa fa-linkedin-square"></i></a></li>
+                        <li><a href="#"><i class="icon-item fa fa-envelope-o"></i></a></li>
+                        <li><a href="#"><i class="icon-item fa fa-github"></i></a></li>
+                        <li><a href="#"><i class="icon-item fa fa-hand-peace-o"></i></a></li>
+                    </ul>
+                </div>
+                <div class="button">
+                    <button type="button">Get my resume</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="skills">
+            <div class="skills-title">
+                <h3>languages</h3><i class='fa fa-chevron-down'></i>
+            </div>
+            <div class="skills-items">
+                <ul>
+                    <li><img src="img/js.png" alt="javascript logo image">JavaScript</li>
+                    <li><img src="img/html.png" alt="html logo image">HTML</li>
+                    <li><img src="img/css.png" alt="css logo image">CSS</li>
+                </ul>
+            </div>
+            <div class="skills-title not-open">
+                <h3>Frameworks</h3><i class='fa fa-chevron-right'></i>
+            </div>
+            <div class="skills-title not-open">
+                <h3>Skills</h3><i class='fa fa-chevron-right'></i>
+            </div>
```

Comment:
As the project requirement let's use `ul` and `li` for the categories in the "About myself" section.

<img width="997" alt="image" src="https://user-images.githubusercontent.com/34578631/170736564-17d8da4f-8cab-4421-bf03-1c33c6bcb680.png">